.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+    docker:
+    - image: circleci/node:8.10
+    working_directory: ~/singnet/platform-contracts
+    steps:
+    - checkout
+    - run:
+        name: Build platform-contracts
+        command: |
+          npm install ganache-cli
```

Comment:
ganache-cli should not be necessary to build and test platform-contracts solely

Reply:
removing ganache-cli installation leads to the error: Cannot find module 'ethereumjs-abi'

Reply:
It should be installed by ```npm install``` as transitive dependency:
```
singularitynet-platform-contracts@0.2.1 /home/vital/projects/singnet/src/github.com/singnet/platform-contracts
│─┬ truffle-hdwallet-provider@0.0.5 
│ └─┬ web3-provider-engine@14.0.6
│   ├─┬ eth-sig-util@1.4.2 
│   │ └── ethereumjs-abi@0.6.5  (git+https://github.com/ethereumjs/ethereumjs
```

Reply:
I had: UNMET DEPENDENCY ethereumjs-abi

├─┬ truffle-hdwallet-provider@0.0.5
│ └─┬ web3-provider-engine@14.0.6
│   ├─┬ eth-sig-util@1.4.2
│   │ ├── UNMET DEPENDENCY ethereumjs-abi@git+https://github.com/ethereumjs/ethereumjs-abi.git
│   │ └─┬ ethereumjs-util@5.2.0

Reply:
Can you try install it as devDependency? (ganache-cli I mean)

Reply:
Adding ganache-cli as dev dependency may be useful to not install it each time. 
But the question is why one npm instance resolves dependencies correctly and another doesn't.

Reply:
The unmet dependencies with ethereumjs-abi are reproduced with npm 5.6.0 which is shipped with the docker image circleci/node:8.10.
I tried the latest npm 6.4.1 on the same environment and ethereumjs-abi is properly installed.